integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiAuxiliaryImage.java

```diff
@@ -195,6 +201,10 @@ public void testCreateDomainUsingMultipleAuxiliaryImages() {
     // create models dir and copy model, archive files if any
     Path modelsPath2 = Paths.get(multipleAIPath2.toString(), "models");
     assertDoesNotThrow(() -> Files.createDirectories(modelsPath2));
+    Path multipleAIPathToFile2 = Paths.get(RESULTS_ROOT, "multipleauxiliaryimage2/test.txt");
+    String content2 = "2";
+    assertDoesNotThrow(() -> Files.write(multipleAIPathToFile2, content2.getBytes()),
```

Comment:
Can you delete the file if it exists?

Reply:
we always delete whole directory before creating anything, see below code:
assertDoesNotThrow(() -> FileUtils.deleteDirectory(multipleAIPath2.toFile()));
    assertDoesNotThrow(() -> Files.createDirectories(multipleAIPath2));

    // create models dir and copy model, archive files if any
    Path modelsPath2 = Paths.get(multipleAIPath2.toString(), "models");
    assertDoesNotThrow(() -> Files.createDirectories(modelsPath2));
    Path multipleAIPathToFile2 = Paths.get(RESULTS_ROOT, "multipleauxiliaryimage2/test.txt");

Reply:
lg